zebedee-cms/src/test/java/com/github/onsdigital/zebedee/permissions/cmd/CMDPermissionsServiceImplTest.java

```diff
@@ -701,4 +703,92 @@ public void testGetUserInstancePermissions_canEditIOEx() throws Exception {
             throw ex;
         }
     }
+
```

Comment:
Might be worth adding annotations stating what each test is testing - saving the reader for having to read the function to find out.